cmd/ctl/pkg/check/api/api.go

```diff
@@ -32,6 +32,7 @@ import (
 	"k8s.io/kubectl/pkg/util/i18n"
 	"k8s.io/kubectl/pkg/util/templates"
 
+	cmcmdutil "github.com/jetstack/cert-manager/cmd/util"
```

Comment:
nitpick: we're obviously in a slightly tricky situation with naming here because we have both `"k8s.io/kubectl/pkg/cmd/util` and `github.com/jetstack/cert-manager/cmd/util`

could we maybe rename `cmdutil "k8s.io/kubectl/pkg/cmd/util"` to `k8scmdutil` just to really be clear about the two different import paths?